jwst/tests_nightly/general/miri/test_sloperpipeline_1.py

```diff
@@ -25,9 +25,11 @@ def test_detector1pipeline1(_bigdata):
     step.refpix.odd_even_rows = True
     step.jump.rejection_threshold = 250.0
     step.ramp_fit.save_opt = False
+    step.output_file='jw00001001001_01101_00001_MIRIMAGE'
+    step.suffix='rate'
```

Comment:
You probably don't even need these two lines (28-29), because they're setting the output file name to exactly the same name that the pipeline infrastructure would use on its own. But it certainly doesn't hurt to explicitly force it.